src/app/aws-exports.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  Auth: {
+  // REQUIRED - Amazon Cognito Identity Pool ID
+      identityPoolId: 'us-east-2:a474fd65-5d53-4a95-8c88-62b800b8fbaf', 
+  // REQUIRED - Amazon Cognito Region
+      region: 'us-east-2',
+      userPoolId: 'us-east-2_1b7twTyIS',
+      userPoolWebClientId: '32qk7q3p62vpf85op2omg9v3n8'
```

Comment:
I'm assuming all these values are okay to be shared publicly?

Reply:
I think so. It's all information that must to be served to the client so that it can access AWS. None of it is secret.

Reply:
Just a thought, but do we prefer storing these values in the repo, or as environment variables or something in the build environment (i.e. currently our machines)?
There''s no security implication if they're client-side values anyway, just wouldn't necessarily want to have to make a commit if we just want to e.g. deploy to a different region.

If this is the standard way though, I'm sure this is fine.